Loadouts/blu_usmc_m16_desert.hpp

```diff
@@ -0,0 +1,259 @@
+//Author: BWMF
+//Description: USMC: M16A4 - Desert Camo
+
+class rhs_faction_usmc_wd {
+  //Rifle
+  #define WEST_RIFLE "rhs_weap_m16a4_carryhandle"
+  #define WEST_RIFLE_MAG "rhs_mag_30Rnd_556x45_Mk318_Stanag:8","rhs_mag_30Rnd_556x45_M855A1_Stanag_Tracer_Red:2"
+  #define WEST_RIFLE_ATTACHMENTS "rhsusf_acc_anpeq15A"
+  #define WEST_AAR_ATTACHMENTS WEST_RIFLE_ATTACHMENTS
+  //GL Rifle
+  #define WEST_GLRIFLE "rhs_weap_m16a4_carryhandle_M203"
+  #define WEST_GLRIFLE_MAG WEST_RIFLE_MAG
+  #define WEST_GLRIFLE_MAG_SMOKE "rhs_mag_m714_White:2","rhs_mag_m713_Red:2"
+  #define WEST_GLRIFLE_MAG_HE "rhs_mag_M433_HEDP:5"
+  #define WEST_GLRIFLE_MAG_FLARE "rhs_mag_M585_white:2","rhs_mag_m661_green:2","rhs_mag_m662_red:2"
+
+  // GL Launcher
+  #define WEST_GLLAUNCHER "rhs_weap_m32"
+  #define WEST_GLLAUNCHER_MAG_HE "rhsusf_mag_6Rnd_M441_HE:3"
+  #define WEST_GLLAUNCHER_MAG_SMOKE "rhsusf_mag_6Rnd_M714_white:5"
+  //Carbine
+  #define WEST_CARBINE WEST_RIFLE
+  #define WEST_CARBINE_MAG WEST_RIFLE_MAG
+  // AR
+  #define WEST_AR "rhs_weap_m249_pip_L"
+  #define WEST_AR_MAG "rhs_200rnd_556x45_M_SAW:2"
+  // AT
+  #define WEST_AT "rhs_weap_rpg26"
+  #define WEST_AT_MAG "rhs_rpg26_mag"
+  // MAT
+  #define USMC_MAT "rhs_weap_smaw_green"
+  #define USMC_MAT_MAG "rhs_mag_smaw_HEDP:2", "rhs_mag_smaw_SR:5"
+  #define USMC_MAT_MAG2 "rhs_mag_smaw_HEAA:2"
+  #define USMC_MAT_OPTIC "rhs_weap_optic_smaw"
+  // HAT
+  #define WEST_HAT "rhs_weap_fgm148"
+  #define WEST_HAT_MAG "rhs_fgm148_magazine_AT:2"
+  // MMG
+  #define WEST_MMG "rhs_weap_m240G"
+  #define WEST_MMG_MAG "rhsusf_100Rnd_762x51:2"
+  #define WEST_MMG_MAG2 "rhsusf_100Rnd_762x51:4"
+  // Sniper Rifle
+  #define WEST_SNIPER "srifle_DMR_06_olive_F"
+  #define WEST_SNIPER_MAG "20Rnd_762x51_Mag:4", "ACE_20Rnd_762x51_Mag_Tracer:4"
+  // Spotter Rifle
+  #define WEST_SPOTTER "rhs_weap_m4a1_blockII_grip2_KAC"
+  #define WEST_SPOTTER_MAG "rhs_mag_30Rnd_556x45_Mk318_Stanag:8"
+  // SMG
+  #define WEST_SMG "hlc_smg_mp5a4"
+  #define WEST_SMG_MAG "hlc_30Rnd_9x19_B_MP5:6"
+  // Pistol
+  #define WEST_PISTOL "rhsusf_weap_m9"
+  #define WEST_PISTOL_MAG "rhsusf_mag_15Rnd_9x19_FMJ:3"
+
+  class Car {
+    TransportWeapons[] = {WEST_AT};
+    TransportMagazines[] = {WEST_RIFLE_MAG,WEST_RIFLE_MAG,WEST_CARBINE_MAG,WEST_AR_MAG,WEST_AR_MAG,WEST_GLRIFLE_MAG_HE};
+    TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+  };
+  class Tank {
+    TransportWeapons[] = {WEST_AT};
+    TransportMagazines[] = {WEST_RIFLE_MAG,WEST_RIFLE_MAG,WEST_CARBINE_MAG,WEST_AR_MAG,WEST_AR_MAG,WEST_GLRIFLE_MAG_HE};
+    TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+  };
+  class Helicopter {
+    TransportMagazines[] = {WEST_RIFLE_MAG,WEST_RIFLE_MAG,WEST_CARBINE_MAG,WEST_AR_MAG,WEST_AR_MAG,WEST_GLRIFLE_MAG_HE};
+    TransportItems[] = {BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL,BASE_MEDICAL};
+  };
+  class Plane {};
+  class Ship_F {};
```

Comment:
https://github.com/BourbonWarfare/bwmf/blob/master/Loadouts/blu_us_m4_ocp.hpp#L55
Sure, but all of the other loadouts have that.